### ISSUE TITLE
Try out a Vagrant approach to the dev setup

### ISSUE THREAD

**Issue by nwinter** [codecombat/codecombat#2106]
Moving from #1241. This might be easier than some of the stuff we're doing with devSetup.py and the separate Windows installer.

**Comment by ssfinney** [comment]
Is [this project](https://github.com/dpen2000/Codecombat-Vagrant) the solution for this? Or were you looking for something else?

**Comment by nwinter** [comment]
I think so, but I haven't tested. It would be good if we test and then move that project into the main CodeCombat repository (at least, from my scanty knowledge of how Vagrant works). Right?

**Comment by ssfinney** [comment]
I'm testing it now, but I'm having some trouble. I'll mess with it and see if I can get it working.

**Comment by AlanHohn** [comment]
I've tested it, and with a few changes (pull requests pending) I was able to get it running, at least as far as seeing the title screen in the browser. (Which is a credit to dpen2000 who made the thing.)

Moving that project into the main repository would be pretty easy and would actually simplify it some. As long as the Vagrantfile is at the top level, all of the various subdirectories in the codebase would be visible under /vagrant in the VM.

Happy to make a pull request for this if there's interest.
